src/App.jsx

```diff
@@ -4,6 +4,8 @@ import Home from "./pages/home"
 import Signin from "./pages/signin"
 import Register from "./pages/register"
 import Player from "./pages/player"
+import Nowplaying from "./pages/nowplaying"
```

Comment:
O nome não deveria ser em camel case?
```suggestion
import NowPlaying from "./pages/nowplaying"
```